dagster/vendor_images.yaml

```diff
@@ -3,7 +3,7 @@ docker.io:
     postgres:
       - "9.6.24"
     dagster/user-code-example:
-      - "1.3.2"
+      - "1.3.5"
```

Comment:
this might not be needed anymore because of the new https://github.com/pluralsh/containers repository?